feature/qos/ate_tests/one_sp_queue_traffic_test/one_sp_queue_traffic_test.go

```diff
@@ -777,38 +777,70 @@ func ConfigureQoS(t *testing.T, dut *ondatra.DUTDevice) {
 	d := &oc.Root{}
 	q := d.GetOrCreateQos()
 
+	type qosVals struct {
+		be0, be1, af1, af2, af3, af4, nc1 string
+	}
+
+	qos := qosVals{
+		be0: "BE0",
+		be1: "BE1",
+		af1: "AF1",
+		af2: "AF2",
+		af3: "AF3",
+		af4: "AF4",
+		nc1: "NC1",
+	}
+
+	if dut.Vendor() == ondatra.JUNIPER {
+		qos = qosVals{
+			be0: "6",
+			be1: "0",
+			af1: "4",
+			af2: "1",
+			af3: "5",
+			af4: "2",
+			nc1: "3",
+		}
+
+	}
+
+	nc1InputWeight := uint64(200)
+	if deviations.SchedulerInputWeightLimit(ondatra.DUT(t, "dut")) {
```

Comment:
Please reuse the dut variable which is available in this function.

i.e `deviations.SchedulerInputWeightLimit(dut)`